### ISSUE TITLE
Choosing "Reconcile->Actions->Create a new topic for cell" results in no output for Wikidata QuickStatements

### ISSUE THREAD

**Issue by SannitaSSJ** [OpenRefine/OpenRefine#1663]
#### Version of OpenRefine used (Google Refine 2.6, OpenRefine2.8, an other distribution?):
v. 3.0 beta

#### Operating Systems and version:
Windows 10

#### Browser + version used:
Chrome 67

#### Steps followed to create the issue:
This issue is close to #1608 .

While reconciliating data to Wikidata, I tagged ~320 cells as "new topic", ~25 cells manually and the others with command "Reconcile > Actions > Create a new item for each cell".

I also created a Wikidata schema in order to populate the new items to be created:
![35543015_10215704538462090_4813365527890100224_n](https://user-images.githubusercontent.com/31443821/41813911-e15aca30-773f-11e8-9b26-f09d991a1193.jpg)

#### Current Results:
While exporting the results to Wikidata Quickstatements, OpenRefine gave the following error messages:
> No edit was generated.

and

> Invalid identifier space for reconciled cells. Some reconciled cells such as $name were ignored because they are not reconciled to Wikidata.

In other words, all cells tagged as "new topic" with the command "Create a new item for each cell" did not produce any command for creating a new item on Wikidata.

#### Expected Results:
To be able to use the Wikidata schema also when marking cells as "new topic" with that command, instead of being forced to manually mark every single cell as such.

**Comment by yarl** [comment]
Same here, also Win 10, 3.0 beta.

**Comment by wetneb** [comment]
@yarl thanks for the report. Can you try to reproduce this with the latest version from git?
Instructions to run the development version are here: https://github.com/OpenRefine/OpenRefine/wiki/Installation-Instructions#development-version

**Comment by wetneb** [comment]
@yarl @SannitaSSJ this bug _should_ be been solved in the latest release (3.0) - it would be great if you could let me know if you still encounter the issue.

**Comment by SannitaSSJ** [comment]
@wetneb it seems to be working now, thanks :) I'll do some more tests and let you know asap

**Comment by wetneb** [comment]
Closing this as it appears to be solved.
